eos-tech-support/eos-usb-internet-writer

```diff
@@ -0,0 +1,229 @@
+        path="../.kiwix/wikipedia_en_endless_maxi.zim"
+        title="EndlessOS selection for English encyclopedia app"
+        description="EndlessOS selection for English encyclopedia app"
+        language="eng" creator="Wikipedia" publisher="Kiwix"
+        name="wikipedia_en_endless" flavour="_maxi"
+        tags="wikipedia;_category:wikipedia;_pictures:yes;_videos:no;_details:yes;_ftindex:yes"
+        date="2020-07-01" articleCount="50001" mediaCount="457288" size="6402513"
+        faviconMimeType="image/png"
+        favicon="iVBORw0KGgoAAAANSUhEUgAAADAAAAAwCAYAAABXAvmHAAAACXBIWXMAAAsSAAALEgHS3X78AAAFVklEQVRo3u1Ye6hlUxhfe+9zb3krEhOihm40MqOUyCMNkiJM5DZmTBh37fO65wwZUR7hPxnmDw1p8IeIlMdoGv7gDnFRZiQRNRFGHmXIPO7M3n7ft761z7f32Wcy5/4xV+1Vv9be31l7re/9fesYU41qVKMa1ahGNarx/xlp7OYkds/+3c9zl3GbYz7Aeyj0gGkr56gQ6YRjXlkgUs+jzPzEbA5o9Q5gk1rWTkQH0ZxAU0LrrbMHrnXeQ5jHfAje12LeDowLLRzKAolVB4hZ9+O/kQjnmJtwDJL5+xhXwsJlamqPy4HvoLg0vduk+O0tWRcN7UIkvTp8PnA1aDdhvg64ADhhgDChZlQznjHV+/1YYH3aAONNoA3E5jMIcJrsFyR2FsxjPgZ4GYfs5AMmgRYfQvgD9I+Bx4Er8H5YQeMRBWbSyKwa5nzdmhWgbWem3Z77gIcgTOCZ15nIZycf7Oxe4sbK1ZUb1c0oaSPtygGecS9IV9ABGvzbNmywDkxdkt7Wp/Gaej8PTGzm/UTr/B6bhSUKJIZDb9lSZTfE8sXUi4cWM0dMW/MucCVwLnARaMsxP4F5GtjJAnUEdV6/BfS7gHmK8VOB5/j3lriLNb8DE2pNLZc0CpkIwpBST4KSFmJehDUnqrgN8xaIzUZmyJrvKUPsJ+DHsKaDNR8Ae/kbb5nY/AWsxe+PsBt6F3QWW+/jSJiOJOtEBabHQGsAr+D5K6z7E9iD9z30jPOnQb9G1RLejKT5PF3FArwhh4wIPRSTRemyvkyzCPQH2ALWzLC221lwemtO4dAzFYMjEi+hcqHjQLOYp4BdLPgqQUdc2Fu9Lc/wmMz94P8jeNgqArwmB4Xik/naYHmtLkRHgcF1wvDuLMPErDXab5ozWYlVQT87dd/+xowpt8S534L+EnAv9l+OeRnWPol5h1g0AcZ0JppiV4AllLvUfCErHBzigEvxzYscEz5AnQtuBd7L/L6VaYwYmiSBgXOw5lUWsKOynTU/iAtejPnQAYXxev7O7fmg9r3HhEi4veTDI4DzgYeBL9hdvGmdxonxFT4D4X0xBH+bteXh4uSXLHachXax8qy5BTiyqCixOrlwTao6ecY37C2x8xZvgdOBGWFmHxZTqnsGeBbYhPdtzHRHac2a3WByQ+KKnj44UIqZzwHf4PX/8B4kgGN8L9AtMD1KSFRbkfbc2BU72s8J8E6uucKCGzKzdlUg3SmzY3qG64U19wNnFIuZBLzP64eDthr4iX1W1QJVzKhAbgSWUiEt7OfjLhJ39vtuIX4gyJtl1XgB8BTwqfjt18CHYgmLTc4qpD1feIJCAbuRi91kLjO9n1LsWLOSXa6pUrAT6Gfgefy+BJg3oGu4kD2lwwKsGdgP+VYXJo8GBFMW3IWsRHViQ1bNXQGjfL6khJnFoL/Ql4Va0rqQq7gstRpoAmt4bV0siSrff9HwgRPnb1C+HiQuAw3q7e/hPN7KNP43vruPaorvd3w3WxDkeOx5B+ibOE22lAt3C2jLHJtHc0rH4uKdwGR1QAtY77XgSa8BozL/CW/ezLISpclTCp1rXillwlhzMjDObmzNR8CPwA5Rxq/Sztys2/Zhr4befW7Nmr62NHmxubbY7+QU0+jrOgMJ/qDkrKMTy5lsTLf0cjma1d02kOcv6VIi6fFpyjxZDvdC0qWn3q8tBHjxxub7m0jajfLraVk3ekA3uFhdxGNzGZ5fB65S7lEr5vD/pJhmvxsnLrsF2X2A6ONu7VCXn6Il+ipnzzJz92+SJM7dd0Mp7+EwWj/of07py/3QWaEa1ahGNapRjWpU4+CNfwFKm+jnNVvQhQAAAABJRU5ErkJggg=="
+    />
+</library>
+XML
+)
+
+run_kolibri() {
+    kolibri_home=$(realpath "$tmp_dir/workspace/.kolibri")
+    KOLIBRI_HOME="$kolibri_home" flatpak run \
+        --filesystem="$kolibri_home" \
+        --command=/app/libexec/kolibri \
+        org.learningequality.Kolibri "$@"
+}
+
+# Install required Flatpak apps
+apps_to_install=(
+    "com.endlessm.encyclopedia.en"
+    "org.learningequality.Kolibri"
+)
+echo "Installing apps ${apps_to_install[*]}. Make sure you have the Flatpak repo installed for those"
```

Comment:
I don't know if we should be installing apps to the host running this script. Maybe just check if they are installed and if not, print an error telling the user they should install / update these flatpaks before running the script.